docs/docsource/operation/Orbit_UI/Bucket_Management/lifecycle_management/index.rst

```diff
@@ -32,7 +32,8 @@ however, the following transition rules can only be defined using API calls.
 * Setting the Days field to zero
 * Setting a custom ID
 
-See :version-ref:`Bucket Lifecycle Operations<http://documentation.scaity.com/Zenko/{version}/reference/cloudserver/bucket_operations/bucket_lifecycle_operations/index.html>` in :version-ref:`Zenko Reference<http://documentation.scaity.com/Zenko/{version}/reference/index.html>`
+See :version-ref:`Bucket Lifecycle Operations <http://documentation.scality.com/Zenko/{version}/reference/cloudserver/bucket_operations/bucket_lifecycle_operations/index.html>` in :version-ref:`Zenko Reference 
```

Comment:
I couldn't fix those broken links (I just changed "scaity" into "scality" but they are still broken). Do you have any clue @wabernatScality?

Reply:
Thanks for your diligence! 

Leave it as it was originally. The way the version-ref thing works is that it looks for where this document is going to be published... which doesn't exist yet. In this instance {version} will be filled in with "2.0.0". This won't break the build. 

This is the way I like to it. Lucie prefers a text reference to the heading in the title. 

The fundamental problem is: how do you drop a live cross-reference into a PDF that crosses the "book boundary"? The answer: You can't. So here we are in Kludgeistan.